takoyaki/src/main/java/frc/robot/subsystems/Climb/FalconBrakeModeClimb.java

```diff
@@ -14,145 +13,96 @@
 
 import com.ctre.phoenix.motorcontrol.ControlMode;
 import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.SupplyCurrentLimitConfiguration;
 import com.ctre.phoenix.motorcontrol.TalonFXInvertType;
 
 public class FalconBrakeModeClimb extends Climb {
 
   private final WPI_TalonFX left;
   private final WPI_TalonFX right;
+  private double setpoint;
   private boolean closedLoop;
-  private boolean goingUp;
-  private boolean separate;
 
   public FalconBrakeModeClimb() {
-
-    closedLoop = false;
-    separate = false;
-
     left = new WPI_TalonFX(Constants.kClimb.LEFT_MOTOR_CAN_ID);
     right = new WPI_TalonFX(Constants.kClimb.RIGHT_MOTOR_CAN_ID);
 
+    left.config_kP(0, Constants.kClimb.kP);
+    left.config_kD(0, Constants.kClimb.kD);
+
     motorConfig(left);
     motorConfig(right);
 
-    // right.follow(left);
+    right.follow(left);
```

Comment:
i am not the most sure about using followers bc of mechanical differences between the climbers but we can see how this works